core/osrd-railjson/src/main/java/fr/sncf/osrd/railjson/schema/rollingstock/RJSEtcsBrakeParams.java

```diff
@@ -9,39 +9,45 @@
  */
 public class RJSEtcsBrakeParams {
 
+    /** National Default Value: Available Adhesion */
+    private final double mNvavadh = 0.0;
+
+    /** National Default Value: Emergency Brake Confidence Level in % */
```

Comment:
As this is not expressed as percentage here:
```suggestion
    /** National Default Value: Emergency Brake Confidence Level ratio */
```

Reply:
Fixed in https://github.com/OpenRailAssociation/osrd/pull/10014/commits/c1bc4feb0ae2e7792d1eb8b8c9171b225d546229